final-progII/src/presentation/Menu.java

```diff
@@ -21,113 +24,120 @@ public Menu(JPanel panel, JPanel master__panel, CardLayout master__cardLayout) {
 		JMenu menu_Absence = new JMenu("Absence");
 		JMenu menu_Exam = new JMenu("Exam");
 		JMenu menu_Exit = new JMenu("Exit");
-		JMenu menu_Space = new JMenu("                                    ");
-		JMenu menu_LogOut = new JMenu("Log Out");
 
 		menuBar.add(menu_User);
 		menuBar.add(menu_Subject);
 		menuBar.add(menu_Absence);
 		menuBar.add(menu_Exam);
 		menuBar.add(menu_Exit);
-		menuBar.add(menu_Space);
-		menuBar.add(menu_LogOut);
 
-		/*Sub Menu of the JMenu Users*/
+		JMenuItem exit__menuItem = new JMenuItem("Menu");
+		menu_Exit.add(exit__menuItem);
+
+		/* Sub Menu of the JMenu Users */
 		JMenuItem createUser__menuItem = new JMenuItem("Create New User");
 		menu_User.add(createUser__menuItem);
-		
+
 		JMenuItem consultUsers_menuItem = new JMenuItem("Consult Users");
 		menu_User.add(consultUsers_menuItem);
-		
+
 		JMenuItem listUsers_menuItem = new JMenuItem("List Users");
 		menu_User.add(listUsers_menuItem);
-		
-		/*Sub Menu of the JMenu Subject*/
+
+		/* Sub Menu of the JMenu Subject */
 		JMenuItem createSubject_menuItem = new JMenuItem("Create Subject");
 		menu_Subject.add(createSubject_menuItem);
 		JMenuItem consultSubject__menuItem = new JMenuItem("Consult Subjects");
 		menu_Subject.add(consultSubject__menuItem);
-		
+
 		JMenuItem listSubject_menuItem = new JMenuItem("List Subjects");
 		menu_Subject.add(listSubject_menuItem);
-		
-		/*Sub Menu of the JMenu Exams*/
+
+		/* Sub Menu of the JMenu Exams */
 		JMenuItem createExan_menuItem = new JMenuItem("Create Exam");
 		menu_Exam.add(createExan_menuItem);
-		
-		/*Sub Menu of the JMenu Absences*/
+
+		/* Sub Menu of the JMenu Absences */
 		JMenuItem createAbsence_menuItem = new JMenuItem("Create Absences");
 		menu_Absence.add(createAbsence_menuItem);
-		
+
 		JMenuItem consultAbsence_menuItem = new JMenuItem("Consult Absences");
 		menu_Absence.add(consultAbsence_menuItem);
-		
+
 		JMenuItem listAbsence_menuItem = new JMenuItem("List Absences");
 		menu_Absence.add(listAbsence_menuItem);
 
+		ControllerLogic c = new ControllerLogic();
```

Comment:
Why you are using that here? It could be easier to read if you declare it at the top of the class

Reply:
Okey .